### PR TITLE
[WIP] Skip if order noncheckable. Compare to the ``sort()`` order

### PR DESCRIPTION
**Pull request by jaryn**
__Fixing__ ordering check. Compare with the order of python ``sort()``. If ordering cannot be checked due to environment state (all items are the same), the test will be skipped.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 9554058a9e9d0dcd51a64c8a92b3da49fc0da0c7:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 667c84e1c2654397829374e0ee7da2e683109617:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/containers/test_containers_main_pages_sort.py [hunk 1]
```diff
@@ -44,4 +43,13 @@ def test_containers_main_pages_sort(cls):
         sort_tbl.sort_by(header_text, 'descending')
         rows_descending = [r[col].text for r in sort_tbl.rows()]
 
-        assert rows_ascending[::-1] == rows_descending
+        expected_ascending_order = sorted(rows_ascending)
+        expected_descending_order = expected_ascending_order[::-1]
+
+        if expected_ascending_order == expected_descending_order:
```
Comment (RonnyPfannschmidt) [inline cfme/tests/containers/test_containers_main_pages_sort.py]: this description is most confusing, as the issue is, that with 0 or 1 items the test hasn't met a precondition

this should either be a failure or a more descriptive skip

Comment (RonnyPfannschmidt) [inline cfme/tests/containers/test_containers_main_pages_sort.py]: sorry for the misleading comment, i formed the it from just the surrounding code, missing the case of a number of equal elements

**Comment by mfalesni** [comment]
I don't think that python sorting has the same sorting rules like cfme does as I saw that in some places the letter case does not matter, or leading dot is ignored ...

**Comment by psav** [comment]
So we need to ensure our sorting order is the same, somehow

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mfalesni** [comment]
@psav We could possibly throw the strings in a json, shove it in the rails console on the appliance and ask developers which sorting methods are used. Then we read the JSON back. :)

**Comment by jkrocil** [comment]
Is this still under active development? If not, please close the PR.

**Comment by gshefer** [comment]
@jaryn, ^^^

**Comment by jkrocil** [comment]
Feel free to reopen once you spend some time on this again. Thanks!

**Pull request closed without merging**